internal/relationtuple/uuid_mapping_test.go

```diff
@@ -321,4 +321,31 @@ func TestMapper(t *testing.T) {
 			})
 		}
 	})
+
+	t.Run("suite=ro and rw mappers", func(t *testing.T) {
+		roMapper := reg.ReadOnlyMapper()
+		rwMapper := reg.Mapper()
+
+		t.Run("case=ro manager does not insert into DB", func(t *testing.T) {
+			unmapped := &ketoapi.RelationQuery{Object: pointerx.Ptr("unmapped object")}
+			mapped, err := roMapper.FromQuery(ctx, unmapped)
+			require.NoError(t, err)
+			assert.NotNil(t, mapped.Object)
+
+			reUnmapped, err := roMapper.ToQuery(ctx, mapped)
+			require.NoError(t, err)
+			assert.Empty(t, reUnmapped.Object)
```

Comment:
Hmm, maybe this case should instead return an error, as the RO mapper is unable to do the mapping? Just to not have bugs later on.